doc/sphinx/language/coq-library.rst

```diff
@@ -281,19 +279,24 @@ In the basic library, we find in ``Datatypes.v`` the definition
 of the basic data-types of programming,
 defined as inductive constructions over the sort ``Set``. Some of
 them come with a special syntax shown below (this syntax table is common with
-the next section :ref:`specification`):
-
-.. productionlist::
-   specif :  `specif` * `specif`                           (prod)
-          : `specif` + `specif`                          (sum)
-          : `specif` + { `specif` }                      (sumor)
-          : { `specif` } + { `specif` }                  (sumbool)
-          : { `ident` : `specif` | `form` }              (sig)
-          : { `ident` : `specif` | `form` & `form` }       (sig2)
-          : { `ident` : `specif` & `specif` }             (sigT)
-          : { `ident` : `specif` & `specif` & `specif` }    (sigT2)
-  term : (`term`, `term`)                               (pair)
-
+the next section :ref:`specification`).  The constructs for :production:`specif` are:
+
+============================================= =======
+:n:`@specif * @specif`                        prod
+:n:`@specif + @specif`                        sum
+:n:`@specif + { @specif }`                    sumor
+:n:`{ @specif } + { @specif }`                sumbool
+:n:`{ @ident : @specif | @form }`             sig
+:n:`{ @ident : @specif | @form & @form }`     sig2
+:n:`{ @ident : @specif & @specif }`           sigT
+:n:`{ @ident : @specif & @specif & @specif }` sigT2
+============================================= =======
+
+And for :token:`term`:
+
+=================== =======
+:n:`(@term, @term)` pair
+=================== =======
```

Comment:
This table for just one line seems too much. It could be replaced by something like:

>A notation for pairs (elements of type `prod`) is defined: ``` :n:`(@term, @term)` ```